code/modules/hydroponics/trays/tray_soil.dm

```diff
@@ -52,14 +49,16 @@
 	icon = 'icons/obj/seeds.dmi'
 	icon_state = "blank"
 
-/obj/machinery/portable_atmospherics/hydroponics/soil/invisible/New(var/newloc,var/datum/seed/newseed)
+/obj/machinery/portable_atmospherics/hydroponics/soil/invisible/New(var/newloc,var/datum/seed/newseed,var/start_mature)
```

Comment:
should be Initialize() here, though make sure to test it to see if it works

Reply:
implemented and tested, works fine as far as i can see